### ISSUE TITLE
No more codeactions

### ISSUE THREAD

**Issue by KristofKlein** [microsoft/AL#6849]
**1. Describe the bug**
I am no longer able to use AL CodeActions Implement Interface. There are no Code Actions shown. I have checked the setup and Codeactions is enabled.
This must have happened with the latest Version: when I use Version 8.1.525271 or 8.1.540594 they show up! 

**2. To Reproduce**
Create a new Codeunit that implements an interface e.g. Invoice Posting - see if you can get the code actions to show up.

**3. Expected behavior**
CodeActions are visible as before and usable.

**4. Actual behavior**
No Code actions are shown. in Fact the VSCode Log shows me something like this:

```
[2021-11-26 10:21:54.954] [renderer1] [error] Cannot read property 'map' of null: TypeError: Cannot read property 'map' of null
	at asDiagnostics (c:\Users\kkl\.vscode\extensions\ms-dynamics-smb.al-8.2.545335\node_modules\vscode-languageclient\lib\common\protocolConverter.js:34:28)
	at Object.asCodeAction (c:\Users\kkl\.vscode\extensions\ms-dynamics-smb.al-8.2.545335\node_modules\vscode-languageclient\lib\common\protocolConverter.js:598:34)
	at c:\Users\kkl\.vscode\extensions\ms-dynamics-smb.al-8.2.545335\node_modules\vscode-languageclient\lib\common\client.js:1226:75
	at async A.provideCodeActions (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:94:172824)
```

**5. Versions:**

- AL Language:
- 8.2.545335
- Visual Studio Code:
- 1.62.3
- Business Central:
- n/a (but in fact it is for BC 19 - so runtime 8.0)
- List of Visual Studio Code extensions that you have installed:
- AL Extensions

### Final Checklist

Please remember to do the following:

* [x] Search the issue repository to ensure you are reporting a new issue

* [x] Reproduce the issue after disabling all extensions except the AL Language extension

* [x] Simplify your code around the issue to better isolate the problem

**Comment by kalberes** [comment]
Similar to #6844, and #6842

**Comment by thloke** [comment]
Heads up that we'll be releasing a new version of the VSIX that addresses this. The fix will be in version v8.2.550913 (or greater).

**Comment by JesperSchulz** [comment]
The fix for this issue has been checked in to the master branch. It will be available in the bcinsider.azurecr.io/bcsandbox-master Docker image starting from platform build number 33156 and VS Code Extension Version <None>. 
 
 If you don’t have access to these images you need to become part of the Ready2Go program: aka.ms/readytogo 
 
 For more details on code branches and docker images please read: 
 https://blogs.msdn.microsoft.com/nav/2018/05/03/al-developer-previews-multiple-releases-and-github/ 
 https://freddysblog.com/2020/06/25/working-with-artifacts/
